docs/en/dce/features.md

```diff
@@ -0,0 +1,99 @@
+# features
+
+DaoCloud Enterprise 5.0 combines a large number of featuere, and the typical featuers currently supported are as follows. 
+
+=== "[multi-cluster management](../kpanda/03ProductBrief/WhatisKPanda.md)"
+
+- Unified management of multi-cluster, supporting any Kubernetes cluster within all specific version range to be included in the container management scope, 
+
+- Rapid cluster deployment. Rapidly deploys enterprise-level Kubernetes cluster on the Web UI, quickly builds enterprise-level container cloud platform, and adapt to the underlying environment of physical machines and VMS. 
+
+- One-click cluster upgrade. Upgrade the Kubernetes version with one click, and manage the upgrade of system components. 
+
+- Cluster high avalability,the built-in cluster disaster recovery and backup abilities ensure that business system can recover from host failures, equipment room interruptions, and natural disasters, improving the stability of the production environment and reducing the risk of business interruption.
+
+- Full lifecycle management of clusters, realizing the full lifecycle management of self-build cloud native clusters. 
+
+- Open API ability, providing native Kubernetes Open API ability. 
+
+=== "Cloud native application management"
+
+- Multi-form cloud native application management, realizing the deployment and lifecycle management of different types of cloud native application, supporting cloud native application, Helm, OEM,  etc., and open multi-cloud application compatibility. 
+
+- One-stop deployment to decouple the underlying Kubernetes platform. One-stop deployment, operation and maintenance of service application to realize the lifecycle-management of application. 
+
+- Expansion and reduction of application load. It can manually/automatically expand and reduce the application load,应用负载的扩缩容，通过界面可实现应用负载的手动/自动扩缩容能力，自定义扩缩容策略，应对流量高峰。
```

Comment:
```suggestion
- Expansion and reduction of application load. It can manually/automatically expand and reduce the application load,应用负载的扩缩容，通过界面可实现应用负载的手动/自动扩缩容能力，自定义扩缩容策略，应对流量高峰。
```

not fully translated